src/main/scala/foo/SlowServer.scala

```diff
@@ -21,8 +22,7 @@ class SlowHttpServer[F[_]: ConcurrentEffect : ContextShift : Timer : Par] extend
 
   def slowRestService() = HttpRoutes.of[F] {
     case GET -> Root / "slow" =>
-      Thread.sleep(SleepTime)
-      Ok("still there?")
+      Timer[F].sleep(SleepTime.millis) *> Ok("still there?")
```

Comment:
We can simulate a slow server without blocking a thread on this server.  If the real "slow server" is blocking, `evalOn` is the answer there.